spec/helpers/simplified_admin_set_selection_work_form_helper_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HykuAddons::SimplifiedAdminSetSelectionWorkFormHelper do
+  include Devise::Test::ControllerHelpers
+
+  let(:user) { create(:user) }
+  let(:model_class) do
+    Class.new(GenericWork) do
+      include ::HykuAddons::GenericWorkOverrides
+    end
+  end
+  let(:work) { model_class.new(title: ["Moomin"], depositor: user.user_key) }
+  let(:form_class) do
+    Class.new(Hyrax::GenericWorkForm) do
+      include ::HykuAddons::GenericWorkFormOverrides
+    end
+  end
+  let(:form) { form_class.new(work, nil, nil) }
+  let(:helper) { _view }
+
+  describe "#form_tabs_for" do
+    context "when the feature is disabled" do
+      before do
+        allow(Flipflop).to receive(:enabled?).with(:simplified_admin_set_selection).and_return(false)
+      end
+
+      it "includes relationships" do
+        expect(helper.form_tabs_for(form: form)).to include("relationships")
+      end
+    end
+
+    context "when the feature is enabled" do
+      before do
+        allow(Flipflop).to receive(:enabled?).with(:simplified_admin_set_selection).and_return(true)
+        allow(helper).to receive(:can_edit?).with(form.model).and_return(true)
+        allow(helper).to receive(:depositor?).with(form.depositor).and_return(true)
+      end
+
+      it "includes relationships" do
+        expect(helper.form_tabs_for(form: form)).not_to include("relationships")
+      end
+    end
```

Comment:
I think it would be good to add another test here for the other permutations `can_edit? = false` and `can_edit? = true && depositor? = false`.